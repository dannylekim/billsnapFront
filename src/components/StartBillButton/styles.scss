@import "../../variables";

button.startBill__button.btn.btn-primary {
  font-family: $primary-font;
  font-size: 15px;
  font-weight: bold;
  background: #007BFF;
  //border: 1px solid #3bd0a3;
  //box-shadow: 0 2px 0 #17c671;
}

button.startBill__button.btn.btn-primary:hover {
  background: #1586FF;
  //border: 1px solid #38b58f;
}
