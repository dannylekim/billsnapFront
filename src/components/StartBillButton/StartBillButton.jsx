import React from "react";
import "./styles.scss";
import {Button} from "shards-react";

export const StartBillButton = () => {
    return (
        <Button size="md" pill className="startBill__button">Start Bill</Button>
    )
};

export default StartBillButton;
