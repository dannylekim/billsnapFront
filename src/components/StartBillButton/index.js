import StartBillButton from "./StartBillButton";

export default StartBillButton;
