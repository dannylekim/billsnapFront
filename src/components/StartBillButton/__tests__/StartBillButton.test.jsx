import React from "react";
import ReactDOM from "react-dom";
import {StartBillButton} from "../StartBillButton.jsx";

describe("StartBillButton", () => {
  describe("render", () => {
    it("renders without crashing", () => {
      const div = document.createElement("div");
      ReactDOM.render(<StartBillButton/>, div);
    });

    describe("snapshots ðŸ“¸", () => {
      it("StartBillButton should match snap shot", () => {
        matches(<StartBillButton/>);
      });
    });
  });
});
