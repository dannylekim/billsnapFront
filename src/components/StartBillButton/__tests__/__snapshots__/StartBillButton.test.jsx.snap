// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StartBillButton render snapshots ðŸ“¸ StartBillButton should match snap shot 1`] = `
<button
  className="startBill__button btn btn-primary btn-md btn-pill"
  onClick={[Function]}
/>
`;
